servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/AlpnClientAndServerTest.java

```diff
@@ -88,19 +88,19 @@ public AlpnClientAndServerTest(List<String> serverSideProtocols,
             "serverAlpnProtocols={0}, clientAlpnProtocols={1}, expectedProtocol={2}, expectedExceptionType={3}")
     public static Collection<Object[]> clientExecutors() {
         return asList(new Object[][] {
-                {asList(HTTP_2, HTTP_1_1), asList(HTTP_2, HTTP_1_1), H2ToStH1Utils.HTTP_2_0, null},
-                {asList(HTTP_2, HTTP_1_1), asList(HTTP_1_1, HTTP_2), H2ToStH1Utils.HTTP_2_0, null},
-                {asList(HTTP_2, HTTP_1_1), singletonList(HTTP_2), H2ToStH1Utils.HTTP_2_0, null},
+                {asList(HTTP_2, HTTP_1_1), asList(HTTP_2, HTTP_1_1), HttpProtocolVersion.HTTP_2_0, null},
```

Comment:
nit: static imports

Reply:
I use static imports for `AlpnIds` for lists. Because of the name conflicts I can not also use static imports for `HttpProtocolVersion`.